readme.md

```diff
@@ -7,14 +7,14 @@
 
```

Comment:
For the `README.md` just fixing some typos, changing terminology here and there.